docker/test/stress/run.sh

```diff
@@ -57,12 +57,12 @@ function start()
         then
             echo "Cannot start clickhouse-server"
             cat /var/log/clickhouse-server/stdout.log
-            tail -n1000 /var/log/clickhouse-server/stderr.log
+            cat /var/log/clickhouse-server/stderr.log | fgrep -v '<Warning> RaftInstance:' -e '<Information> RaftInstance' | tail -n1000
```

Comment:
Do we have `<Warning> RaftInstance` in stderr?

Reply:
You are right, that was the server log, fixed. Thanks.

Reply:
And here is an example of how non informative it looks right now - https://clickhouse-test-reports.s3.yandex.net/26336/11b97258943b2df550fadfab30d44704575db0d0/stress_test_(undefined)/test_run.txt.out.log (because of this raft messages)